src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private int firstSide;
+    private int secondSide;
+
+    public RightTriangle(int firstSide, int secondSide, Color color) {
+        super(color);
+        this.firstSide = firstSide;
+        this.secondSide = secondSide;
+    }
+
+    public void setFirstSide(int firstSide) {
+        this.firstSide = firstSide;
+    }
+
+    public void setSecondSide(int secondSide) {
+        this.secondSide = secondSide;
+    }
+
+    public int getFirstSide() {
+        return firstSide;
+    }
+
+    public int getSecondSide() {
+        return secondSide;
+    }
+
+    @Override
+    public double getArea() {
+        return (firstSide * secondSide / 2);

```

Comment:
The parenthesis in the area calculation are not necessary. It should be 'return firstSide * secondSide / 2.0;' to ensure floating point division.